pkg/tfbridge/info/info.go

```diff
@@ -666,6 +666,27 @@ type Default struct {
 	EnvVars []string
 }
 
+// ComputeDefaultAutonamingOptionsMode is the mode that controls how the provider handles the proposed name. If not
+// specified, defaults to `Propose`.
+type ComputeDefaultAutonamingOptionsMode int32
+
+const (
+	// ModePropose means the provider may use the proposed name as a suggestion but is free to modify it.
+	ModePropose ComputeDefaultAutonamingOptionsMode = 0
```

Comment:
Let's avoid taking the very generic namespace `Mode` for these enums. 

```suggestion
	ComputeDefaultAutonamingModePropose ComputeDefaultAutonamingOptionsMode = 0
```

Also, can we use `iota` here, or is there a reason why not (if not, we need a comment explaining why not).